suites/squid/rados/tier-2_rados_test-pg-dups-trimming.yaml

```diff
@@ -0,0 +1,204 @@
+# Suite is to be used to verify the dups trimming scenario on all NVMe cluster
+# Conf file to be used : conf/squid/baremetal/mero_conf.yaml
+
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: cluster deployment
+      desc: Execute the cluster deployment workflow.
+      module: test_cephadm.py
+      polarion-id: CEPH-83574887
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              base_cmd_args:
+                verbose: true
+              args:
+                # using 6.1 rc candidate
```

Comment:
We can use builds other than 6.1 Here, if additional coverage for other releases needed. otherwise we can continue with the same.

Reply:
I have not made any changes.